distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -445,6 +450,10 @@ grant {
     permission java.io.FilePermission "${java.home}${/}conf${/}jaxm.properties", "read";
 }
 
+grant codeBase "file:/org.jline.terminal" {
```

Comment:
This failure popped up in the itests. The acdebugger stated this as the permission to add, which does make sense to me but I'm unsure why this is now occurring. The only relation I can see with jline and dependencies is zookeeper has jline specified as a dependency.

Reply:
❗️ It ain't a failure and we should not be adding this permission. The latest (not yet released) version of the AC debugger has identified this as an acceptable failure since jline is designed to fallback to a dumb terminal.